### ISSUE TITLE
strange result for npm3d

### ISSUE THREAD

**Issue by amiltonwong** [aboulch/ConvPoint#12]
Hi, @aboulch ,

I had trained npm3d through 99 epochs, and got `91.64` [iou](https://gist.github.com/amiltonwong/d32a05ed58e389d31d1fa749d651b379). For testing, the visualization of the prediction outputs look fine. I attached [ajaccio_2_predict.png](https://www.dropbox.com/s/rb6lcj7rsq7icub/ajaccio_2_predict.png), [ajaccio_57_predict.png](https://www.dropbox.com/s/fhjo3o09c1cgatn/ajaccio_57_predict.png), [dijon_9_predict.png](https://www.dropbox.com/s/afj1mz42nwr4d4e/dijon_9_predict.png) here for reference. However, when I submitted the prediction output to NPM3D benchmark, I got low `mIOU = 30.7`, result [link](https://www.dropbox.com/s/zg3tldc4vbtk9mx/benchmark_result.jpg). It seems that there's strange problem in classifying on Trash can, Barrier, Pedestrian (only get `IOU = 0.x`). And the prediction on Car looks good in visualization but only got `IOU = 19.5` in benchmark. Is  there something conflicted with the original labeling during the [preparation step](https://github.com/aboulch/ConvPoint/tree/master/examples/npm3d#data)? 

THX!

**Comment by aboulch** [comment]
Hello,
Label indices should be OK but I will look into that.
Trash can, Barrier, Pedestrian are difficult classes. Please note that in the paper, results in the paper are obtained with a fusion process that I did not released yet for this dataset.
It may explain the low results on these classes.

**Comment by Dario-Mantegazza** [comment]
Hello @aboulch, first thanks for the interesting paper and approach, I'm trying to use ConvPoint on a custom dataset that is similar to Semantic3D or npm3D

**Comment by aboulch** [comment]
Hello,
I have added pre-trained models on NPM3D and released the fusion code.
The pre-trained models come with no guarantee as I changed affiliation and could not retreive all data from my previous company.
